### ISSUE TITLE
Why can't I open  app? It always fades away

### ISSUE THREAD

**Issue by Khhd1** [floh22/LeagueBroadcastHub#7]
Hey! Bro,Why can't I open  app? It always fades away

**Comment by floh22** [comment]
Im going to need more information.
What Windows version are you using? The App was built against Windows 10 20H1 (May 2020 Update) Build 19041, so if your version is older, it will not work.
What steps did you take to run the app?

**Comment by Khhd1** [comment]
> 我将需要更多信息。
> 您正在使用哪个Windows版本？该应用程序是针对Windows 10 20H1（2020年5月更新）内部版本19041构建的，因此，如果您的版本较旧，则它将无法使用。
> 您采取了哪些步骤来运行该应用程序？

![image](https://user-images.githubusercontent.com/82941693/117573030-65296b00-b108-11eb-8a35-595164e11470.png)
The version of the system I'm using is 20H2, updated in 2021

**Comment by Khhd1** [comment]
> 我将需要更多信息。
> 您正在使用哪个Windows版本？该应用程序是针对Windows 10 20H1（2020年5月更新）内部版本19041构建的，因此，如果您的版本较旧，则它将无法使用。
> 您采取了哪些步骤来运行该应用程序？

My order of use is to download the latest file in Release and then unzip it to my computer. I open the file in LeagueBroadcast.exe, I see the page downloading the latest file and then it disappears

**Comment by floh22** [comment]
Can you send the last log file? There should be a folder called Logs which contains all log files.
Can you also send Farsight.json. It should be in the config folder

**Comment by Khhd1** [comment]
> 您可以发送最后一个日志文件吗？应该有一个名为Logs的文件夹，其中包含所有日志文件。
> 您还可以发送Farsight.json吗？它应该在config文件夹中

![image](https://user-images.githubusercontent.com/82941693/117574485-d6205100-b10f-11eb-8458-15e229e91a8d.png)
I don't have a Farsight. Json

**Comment by floh22** [comment]
It looks like your requests to github are being blocked for some reason. For now, try to manually download https://raw.githubusercontent.com/floh22/LeagueBroadcastHub/v2/Offsets/Offsets-11.9.1.json or https://github.com/floh22/LeagueBroadcastHub/blob/v2/Offsets/Offsets-11.9.1.json, depending on which one works, they are the same.
Call the file Farsight.json, and place it in Config. I will have to add a warning when github cannot be reached

**Comment by Khhd1** [comment]
> 您对github的请求似乎由于某种原因而被阻止。目前，尝试手动下载https://raw.githubusercontent.com/floh22/LeagueBroadcastHub/v2/Offsets/Offsets-11.9.1.json或[https://github.com/floh22/LeagueBroadcastHub/blob/v2/Offsets /Offsets-11.9.1.json](https://github.com/floh22/LeagueBroadcastHub/blob/v2/Offsets/Offsets-11.9.1.json)，取决于哪一个起作用，它们是相同的。
> 调用文件Farsight.json，并将其放在Config中。当无法访问github时，我将不得不添加一个警告

I did it. Thanks, buddy

**Comment by Khhd1** [comment]
> 您对github的请求似乎由于某种原因而被阻止。目前，尝试手动下载https://raw.githubusercontent.com/floh22/LeagueBroadcastHub/v2/Offsets/Offsets-11.9.1.json或[https://github.com/floh22/LeagueBroadcastHub/blob/v2/Offsets /Offsets-11.9.1.json](https://github.com/floh22/LeagueBroadcastHub/blob/v2/Offsets/Offsets-11.9.1.json)，取决于哪一个起作用，它们是相同的。
> 调用文件Farsight.json，并将其放在Config中。当无法访问github时，我将不得不添加一个警告

![image](https://user-images.githubusercontent.com/82941693/117575524-c8b99580-b114-11eb-8523-4ece43ef65f5.png)
I have a suggestion that the top scoreboard of the spectator page could use something like an image, which might call some Riot API, as well as a post-match graph，
![image](https://user-images.githubusercontent.com/82941693/117575598-1f26d400-b115-11eb-9fa5-f5ecfa29fac1.png)

**Comment by Khhd1** [comment]
> > 您对github的请求似乎由于某种原因而被阻止。目前，尝试手动下载[https://raw.githubusercontent.com/floh22/LeagueBroadcastHub/v2/Offsets/Offsets-11.9.1.json或[https://github.com/floh22/LeagueBroadcastHub/blob/v2/Offsets](https://raw.githubusercontent.com/floh22/LeagueBroadcastHub/v2/Offsets/Offsets-11.9.1.json%E6%88%96%5Bhttps://github.com/floh22/LeagueBroadcastHub/blob/v2/Offsets) /Offsets-11.9.1.json]([https://github.com/floh22/LeagueBroadcastHub/blob/v2/Offsets/Offsets-11.9.1.json)，取决于哪一个起作用，它们是相同的。](https://github.com/floh22/LeagueBroadcastHub/blob/v2/Offsets/Offsets-11.9.1.json)%EF%BC%8C%E5%8F%96%E5%86%B3%E4%BA%8E%E5%93%AA%E4%B8%80%E4%B8%AA%E8%B5%B7%E4%BD%9C%E7%94%A8%EF%BC%8C%E5%AE%83%E4%BB%AC%E6%98%AF%E7%9B%B8%E5%90%8C%E7%9A%84%E3%80%82)
> > 调用文件Farsight.json，并将其放在Config中。当无法访问github时，我将不得不添加一个警告
> 
> I did it. Thanks, buddy

My pickban function is not working and has no graphics

**Comment by Khhd1** [comment]
![image](https://user-images.githubusercontent.com/82941693/117577166-3cf73780-b11b-11eb-8956-3690eb0a9526.png)
These features are also not shown in OBS when playing video or watching 5V5, such as being upgraded to level 6

**Comment by floh22** [comment]
Are you using the chinese client? Im pretty sure this tool will not work there because the API and memory locations are all different. Any non riot servers/client will most likely not work as far as I know how the API and Memory locations work.

Otherwise it would be best if you send your log and replay file that you are using to test, so I can try to reproduce the issue

**Comment by Khhd1** [comment]
> 您在使用中国客户吗？我非常确定此工具将无法在此处使用，因为API和内存位置都不同。就我所知道的API和内存位置的工作方式而言，任何非防暴服务器/客户端都极有可能无法正常工作。
> 
> 否则，最好将您要测试的日志和重播文件发送出去，这样我就可以尝试重现该问题
Yes, I am a Chinese client, but in the last  version of the software, I tested LBH-0.3.0 and I can use it normally.

**Comment by Khhd1** [comment]
> > 您在使用中国客户吗？我非常确定的工具将无法在此处使用，因为API和内存位置都不同。就我所知道的API和内存位置的工作方式而言，任何非防暴服务器/客户端都极有可能无法正常工作。
> > 否则，最好
> > 是我是中国客户，但在该软件的最新版本中，我测试了LBH-0.3.0，并且我可以正常使用。

Except for the PICKBAN feature

**Comment by Khhd1** [comment]
[09-43-44] Logging Init. Log Level set to Verbose
[09-43-44] Starting Config Controller
[09-43-44] Config Provider started
[09-43-44] Looking for config files at D:\LeagueBroadcast-1.1.14 (2)\Config
[09-43-44] Found PickBan.json. Reading values
[09-43-45] {
  "frontend": {
    "scoreEnabled": true,
    "spellsEnabled": true,
    "coachesEnabled": true,
    "blueTeam": {
      "name": "TUM",
      "score": 0,
      "coach": "G2 Grabz",
      "color": "rgb(80,140,230)"
    },
    "redTeam": {
      "name": "LMU",
      "score": 0,
      "coach": "G2 Grabz",
      "color": "rgb(239,66,67)"
    },
    "patch": "11.9.1"
  },
  "contentPatch": "latest",
  "contentCdn": "https://ddragon.leagueoflegends.com/cdn",
  "FileVersion": "1.0"
}
[09-43-45] Config PickBan loaded
[09-43-45] Found Component.json. Reading values
[09-43-45] {
  "DataDragon": {
    "MinimumGoldCost": 2000
  },
  "PickBan": {
    "IsActive": true,
    "UseDelay": false,
    "DelayValue": 300
  },
  "Ingame": {
    "IsActive": true,
    "UseLiveEvents": true,
    "DoLevelUp": true,
    "DoItemCompleted": true,
    "Objectives": {
      "DoBaronKill": true,
      "DoDragonKill": true,
      "DoInhibitors": false
    },
    "Teams": {
      "DoTeamNames": false,
      "DoTeamIcons": false,
      "DoTeamScores": false
    }
  },
  "Replay": {
    "IsActive": true,
    "UseAutoInitUI": true
  },
  "PostGame": {
    "IsActive": false
  },
  "App": {
    "LogLevel": 3,
    "CheckForUpdates": true,
    "UpdateRepositoryUrl": "https://github.com/floh22/LeagueBroadcastHub",
    "UpdateRepositoryName": "floh22/LeagueBroadcastHub",
    "LeagueInstall": [
      "C:\\"
    ],
    "CheckForOffsets": true,
    "OffsetRepository": "https://raw.githubusercontent.com/floh22/LeagueBroadcastHub/v2/Offsets/",
    "OffsetPrefix": "Offsets-"
  },
  "FileVersion": "1.0"
}
[09-43-45] Config Component loaded
[09-43-45] League Broadcast (Essence) Version 1.1.14.21129
[09-43-45] Checking for Updates
[09-43-45] [WARNING] Request to https://api.github.com/repos/floh22/LeagueBroadcastHub/releases/latest (TResultType = GitHubReleaseInfo) did return status code Forbidden.
[09-43-45] [WARNING] Could not update
[09-43-45] Early Init Complete in 1424.0268ms
[09-43-45] DataDragon Provider Init
[09-43-45] Getting latest versions from dataDragon
[09-43-45] Retrieving latest patch info
[09-43-46] Champion: 11.9.1, Item: 11.9.1, CDN: https://ddragon.leagueoflegends.com/cdn
[09-43-46] Loaded 155 champions
[09-43-46] Loaded 14 summoner spells
[09-43-46] Detected 222 items
[09-43-46] Loaded 117 full items
[09-43-46] Checking Local Cache
[09-43-46] Cache D:\LeagueBroadcast-1.1.14 (2)/Cache/11.9.1 exists already
[09-43-47] DDragon loaded
[09-43-47] Looking for new Offsets
[09-43-47] Found Farsight.json. Reading values
[09-43-47] {
  "GameVersion": "11.9.1",
  "FileVersion": "1.0",
  "GameOffsets": {
    "Manager": "0x171bbc4",
    "MapCount": "0x2c",
    "MapRoot": "0x28",
    "MapNodeNetId": "0x10",
    "MapNodeObject": "0x14"
  },
  "ObjectOffsets": {
    "ID": "0x20",
    "NetworkID": "0xcc",
    "Team": "0x4c",
    "Pos": "0x1d8",
    "Mana": "0x298",
    "MaxMana": "0x2a8",
    "Health": "0xd98",
    "MaxHealth": "0xda8",
    "CurrentGold": "0x1b50",
    "GoldTotal": "0x1b60",
    "EXP": "0x37ac",
    "Name": "0x304c",
    "ItemList": "0x3714",
    "SpellBook": "0x2ba0",
    "ItemListItem": "0xc",
    "ItemInfo": "0x20",
    "ItemInfoId": "0x68"
  }
}
[09-43-47] Config Farsight loaded
[09-43-47] Loading PickBan Controller
[09-43-47] Loading Ingame Controller
[09-43-47] Checking Default League install for LiveEventsAPI
[09-43-47] [WARNING] Game Location not found. Asking user on startup
[09-43-47] Loading Replay Controller
[09-43-47] Loading Farsight
[09-43-47] Farsight loaded. Found 155 Champ names
[09-43-47] Loading Frontend Webserver (HTTP/WS)
[09-43-47] Server file system starting on: D:\LeagueBroadcast-1.1.14 (2)\cache
[09-43-47] WebServer New State - Loading
[09-43-47] WebServer New State - Listening
[09-43-47] WebServer running on http://localhost:9001/
[09-43-47] Whats that ticking noise?
[09-43-47] Init Complete in 1.292s
[09-43-47] Opening main window
[09-43-47] Connection State Changed
[09-43-47] Loading State Controller
[09-43-47] Connecting to League Client
[09-43-47] PickBan Enabled
[09-43-47] Ingame Enabled
[09-43-47] Using direct PickBan
[09-43-47] Starting Essence with tickrate of 2tps
[09-43-47] Checking for running Game
[09-43-47] Post Init Complete in 223.7127ms
[09-43-47] Total Startup time: 2.940s
[09-43-58] Switch Team View visibility
[09-44-00] Switch Team View visibility

**Comment by Khhd1** [comment]
[WT2_NEW-4141282085.zip](https://github.com/floh22/LeagueBroadcastHub/files/6448936/WT2_NEW-4141282085.zip)
![Uploading image.png…]()

**Comment by floh22** [comment]
This makes sense. Memory structure and security on the Chinese client is completely different to riot clients. Old versions of this project did not use memory reading, so they did not have this issue. 
Sadly I cannot help you since I neither have access to the Chinese client to support it, and if I did, the Chinese client is very very restrictive and locks down memory completely, so i couldn't read memory if I tried.
I'm sorry that the new versions are not usable for you
